### ISSUE TITLE
Fix false failing test

### ISSUE THREAD

**Issue by Govinda-Fichtner** [hypriot/cluster-lab#43]
When the Cluster Lab starts up it often show the following error

```
[FAIL]   Consul is able to talk to Docker-Engine on port 7946 (Serf)
```

while actually everything is working.

So we need to give it more time or to make the test otherwise more reliable.

**Comment by Govinda-Fichtner** [comment]
@firecyberice Any progress here? We need this for Trier.

**Comment by firecyberice** [comment]
solved by PR #47
